.github/workflows/main.yml

```
@@ -16,7 +16,7 @@ jobs:
         numpy_ver: ["latest"]
         test_config: ["latest"]
         include:
-          - python-version: "3.9"
+          - python-version: "3.10"
             numpy_ver: "1.21"
```

```suggestion
            numpy_ver: "2.00"
```

Updating both to current NEP 29 tests: https://numpy.org/neps/nep-0029-deprecation_policy.html